selfdrive/car/honda/hondacan.py

```diff
@@ -167,7 +167,8 @@ def create_ui_commands(packer, CAN, CP, enabled, pcm_speed, hud, is_metric, acc_
     commands.append(packer.make_can_msg("ACC_HUD", CAN.pt, acc_hud_values))
 
   lkas_hud_values = {
-    'SET_ME_X41': 0x41,
+    'ENABLED': 1,
+    'SET_ME_X20': 0x20,
```

Comment:
Does this maintain both MSB 30 and `LDW_OFF` at MSB 27?

Reply:
These are still just for LKAS. I will rename `ENABLED`.